products/oneplus.md

```diff
@@ -0,0 +1,119 @@
+
+-   releaseCycle: "11"
+    releaseDate: 2023-02-07 # https://oxygenupdater.com/article/379/
+    support: 2027-02-07 # approximation "4 major Android updates" https://community.oneplus.com/thread/1211291251581124608
+    eol: 2028-02-07
+    discontinued: false
+    link: https://service.oneplus.com/us/search/search-detail?id=2123192&articleIndex=2
+    supportedAndroidVersions: 13 - 14 # https://oxygenupdater.com/article/426/ https://community.oneplus.com/thread/1465453057260126214
+
+
+# following is a list of phones with their update policy...delete the entry once the device is implemented in the page
+
+# https://community.oneplus.com/thread/1462181 : 3 major android updates and 4 years of security updates
+#    OnePlus 10 Pro
+#    OnePlus 10R
+#    OnePlus 10T
+#    OnePlus Ace
+#    OnePlus Ace Racing Edition
+#    OnePlus 9RT
+#    OnePlus 9 Pro
+#    OnePlus 9R
+#    OnePlus 9
+#    OnePlus 8T
+#    OnePlus 8 Pro
+#    OnePlus 8
+
+# https://community.oneplus.com/thread/1356800969827942405 : 3 major android updates and 4 years of security updates
+#    OnePlus Nord 3
+
+# https://community.oneplus.com/thread/862347 : 2 major android updates and 3 years of security updates
+#    OnePlus 7 Pro
+#    OnePlus 7T Pro
+#    OnePlus 7T
+#    OnePlus 7
+#    OnePlus 6T
+#    OnePlus 6
+#    OnePlus 5T
+#    OnePlus 5
+#    OnePlus 3T
+#    OnePlus 3
+
+# https://community.oneplus.com/thread/1462181 : 2 major android updates and 3 years of security updates
+#    OnePlus Nord CE
+#    OnePlus Nord
+#    OnePlus Nord 2
+#    OnePlus Nord CE 2 Lite
+
+# https://community.oneplus.com/thread/1462181 : 1 major android update and 3 years of security updates
+#    OnePlus Nord N200 5G
+#    OnePlus Nord N100
+#    OnePlus Nord N10 5G
+
+
+---
+
+> OnePlus is a Chinese manufacturer of Android phones and other consumer electronics.
+
+OnePlus phones are running OxygenOS, their own OS that is based on Android.
+They receives updates every two months.
+
+In June 2018, the first update policy [announcement](https://community.oneplus.com/thread/862347) was made offering two major Android updates and three years of security updates for the OnePlus 3/3T, 5/5T, 6 and onwards.
+
+Then in July 2021, OnePlus [announced](https://community.oneplus.com/thread/1462181) that they would now offer three major Android updates and four years of security updates for the flagship line (including T & R series) starting from OnePlus 8 onwards, two major Android updates and three years of security updates for the Nord/Nord CE series, and finally one major Android update and three years of security updates for the Nord N series.
+
+In November 2022, OnePlus [announced](https://community.oneplus.com/thread/1211291251581124608) that they will offer "four generations of OxygenOS" (Android versions) and five years of security updates on "selected devices". In practice this policy will apply only to their flagship lineup (including T & R series) starting from OnePlus 11 onwards.
```

Comment:
+1 to what @marcwrobel suggested:

> Overall that's a good start but I think there is too much information in the description.

Suggestion to implement that:

```suggestion
OnePlus phones run OxygenOS, which is based on Android. It receives updates every two months.

OnePlus supports [four generations of OxygenOS](https://community.oneplus.com/thread/1211291251581124608) (Android versions) and five years of security updates on "selected devices". In practice this policy applies only to their flagship lineup (including T & R series) starting from OnePlus 11 onwards.
```

> this policy applies only to their flagship lineup

It is confusing whether "this policy" means here, in context of Non-Flagship devices:

1. Four generations of OxygenOS are supported but 5 years of security updates are not guaranteed.
2. No support guarantees are made.

Minor note on tone: I changed the continuous tense to present. https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#tone-and-text has more examples.

Reply:
Yes, their update policies are changed and announced all over the place, that's why there's several links. It's a mess. 
In my last force push I used your suggestion (i.e. only mention current update policy and disregard old ones) to cut down on text length, but still mentioned the policies of the other 3 range products (Nord, Nord CE, Nord N).
I can cut even more information but that'll mean completely removing a policy

Reply:
Updated description looks good 👍🏽